content/docs/configuration/issuer-and-clusterissuer-resources/README.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Issuer Configuration
+description: Learn about configuring cert-manager using Issuer and ClusterIssuer resources.
```

Comment:
This description is almost the same as the one fore the configuration readme file.